### ISSUE TITLE
Story: legacy DEDP learners' course certificates 

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#705]
As a returning DEDP learner who earned a course certificate on micromasters.mit.edu, I can link to my past course certificates on my dashboard.

Course certificates will be hosted on mitxonline going forward, so this has a dependency on https://github.com/mitodl/mitxonline/issues/691

Note that the source of truth for DEDP course certificates is the micromasters app. The source of truth for other MIcroMasters programs is edx.org. 

#### Designs and Mockups

same as #691 

#### required MM data 

- [ ] Micromasters course certificates

#### out of scope

- program certificates are a separate story

**Comment by rhysyngsun** [comment]
Tagging this as blocked because we haven't sorted out how to associate certificates with certain signatures.

**Comment by pdpinch** [comment]
Ok, so this is blocked by #887 then.

**Comment by aliraza-abbasi** [comment]
#887 is merged and deployed,
cc: @pdpinch @rhysyngsun

**Comment by rachellougee** [comment]
@pdpinch @rhysyngsun How should we determine which course run to generate the certificate for on MITxOnline?

In MM, the certificate is per course. In MITxOnline, it's per course run

For example, this student has a certificate for "[Data Analysis for Social Scientists"](https://micromasters.mit.edu/admin/grades/micromasterscoursecertificate/1723/change/?_changelist_filters=q%3Dasiroky) in MM
He has passed grades for two course runs on MITxOnline

- [course-v1:MITx+14.310x+2T2020 | Data Analysis for Social Scientists ](https://mitxonline.mit.edu/admin/courses/courserungrade/26458/change/)
- [course-v1:MITx+14.310x+2T2018 | Data Analysis for Social Scientists](https://mitxonline.mit.edu/admin/courses/courserungrade/25518/change/)

When we migrate his certificate, should we generate two certificates (course-v1:MITx+14.310x+2T2020 and course-v1:MITx+14.310x+2T2018), Or should we pick one?

**Comment by pdpinch** [comment]
Oh, I forgot about that. I think Anna reminds me of it frequently. 

We should pick the one with the highest grade, if it's not too much trouble.

**Comment by rachellougee** [comment]
Keep it open until we run it on production

**Comment by rachellougee** [comment]
@pdpinch Please let me know when we should run the migration to bring over the MM certificates.

**Comment by pdpinch** [comment]
This looks good to me. I opened a small issue about spacing in the UI #1206 but it's already been fixed.

**Comment by rachellougee** [comment]
Just run it on production, and the imported number is a little less than I estimated. I think it's due to some learner's grades are not on mitxonline
```
Running: 009_import_course_certificate.sql

Rows added/updated: 4952
```

**Comment by pdpinch** [comment]
Ok. We’ll need to keep track of certificates that exist only on Micromasters, and come up with a plan for them. 

I’ll open an issue for that (somewhere…)

**Comment by rachellougee** [comment]
@pdpinch I can look into these certificates only exist on Micromasters. Just let me know when you want me to do it
